### PR TITLE
T0 add ut for sharding-orchestration-core module

### PR DESCRIPTION
**Pull request by zjcnb**
For  #4452

### CODE REVIEW

File: sharding-orchestration/sharding-orchestration-core/src/test/java/org/apache/shardingsphere/orchestration/internal/registry/config/listener/SchemaChangedListenerTest.java [hunk 1]
```diff
@@ -170,4 +170,42 @@ public void assertCreateSchemaDeletedEventForNewSchema() {
...
+    @Test
+    public void assertCreateWithNullShardingSchemaName() {
+        DataChangedEvent dataChangedEvent = new DataChangedEvent("/test/config/schema/datasource", DATA_SOURCE_YAML, ChangedType.DELETED);
+        ShardingOrchestrationEvent actual = schemaChangedListener.createShardingOrchestrationEvent(dataChangedEvent);
+        assertThat(actual, instanceOf(IgnoredShardingOrchestrationEvent.class));
+    }
+    
+    @Test
+    public void assertCreateAddedEventWithEncryptRuleConfigurationForNewSchema() {
+        DataChangedEvent dataChangedEvent = new DataChangedEvent("/test/config/schema/encrypt_db/rule", ENCRYPT_RULE_YAML, ChangedType.UPDATED);
```
Comment (menghaoranss) [inline sharding-orchestration/sharding-orchestration-core/src/test/java/org/apache/shardingsphere/orchestration/internal/registry/config/listener/SchemaChangedListenerTest.java]: `encrypt_db` is exist schema, the dataChangedEvent will not be created for new schema.

File: sharding-orchestration/sharding-orchestration-core/src/test/java/org/apache/shardingsphere/orchestration/internal/registry/config/listener/SchemaChangedListenerTest.java [hunk 2]
```diff
@@ -170,4 +170,42 @@ public void assertCreateSchemaDeletedEventForNewSchema() {
...
+        ShardingOrchestrationEvent actual = schemaChangedListener.createShardingOrchestrationEvent(dataChangedEvent);
+        assertThat(actual, instanceOf(EncryptRuleChangedEvent.class));
+        assertThat(((EncryptRuleChangedEvent) actual).getEncryptRuleConfiguration(), instanceOf(EncryptRuleConfiguration.class));
+    }
+    
+    @Test
+    public void assertCreateAddedEventWithShardingRuleConfigurationForNewSchema() {
+        DataChangedEvent dataChangedEvent = new DataChangedEvent("/test/config/schema/sharding_db/rule", SHARDING_RULE_YAML, ChangedType.UPDATED);
+        ShardingOrchestrationEvent actual = schemaChangedListener.createShardingOrchestrationEvent(dataChangedEvent);
```
Comment (menghaoranss) [inline sharding-orchestration/sharding-orchestration-core/src/test/java/org/apache/shardingsphere/orchestration/internal/registry/config/listener/SchemaChangedListenerTest.java]: same problem as above

File: sharding-orchestration/sharding-orchestration-core/src/test/java/org/apache/shardingsphere/orchestration/internal/registry/config/listener/SchemaChangedListenerTest.java [hunk 3]
```diff
@@ -170,4 +170,42 @@ public void assertCreateSchemaDeletedEventForNewSchema() {
...
+        assertThat(actual, instanceOf(ShardingRuleChangedEvent.class));
+        assertThat(((ShardingRuleChangedEvent) actual).getShardingRuleConfiguration(), instanceOf(ShardingRuleConfiguration.class));
+    }
+    
+    @Test
+    public void assertCreateAddedEventWithMasterSlaveRuleConfigurationForNewSchema() {
+        DataChangedEvent dataChangedEvent = new DataChangedEvent("/test/config/schema/masterslave_db/rule", MASTER_SLAVE_RULE_YAML, ChangedType.UPDATED);
+        ShardingOrchestrationEvent actual = schemaChangedListener.createShardingOrchestrationEvent(dataChangedEvent);
```
Comment (menghaoranss) [inline sharding-orchestration/sharding-orchestration-core/src/test/java/org/apache/shardingsphere/orchestration/internal/registry/config/listener/SchemaChangedListenerTest.java]: same problem as above

File: sharding-orchestration/sharding-orchestration-core/src/test/java/org/apache/shardingsphere/orchestration/internal/registry/config/listener/SchemaChangedListenerTest.java (hunk 1 above)
Comment (zjcnb) [reply sharding-orchestration/sharding-orchestration-core/src/test/java/org/apache/shardingsphere/orchestration/internal/registry/config/listener/SchemaChangedListenerTest.java]: ok,I'll fix it later

**Pull request merged**